client/post-editor/editor-ground-control/index.jsx

```diff
@@ -170,7 +171,11 @@ export class EditorGroundControl extends PureComponent {
 			const eventLabel = postUtils.isPage( this.props.page )
 				? 'Clicked Preview Page Button'
 				: 'Clicked Preview Post Button';
+			const eventTracksLabel = postUtils.isPage( this.props.page )
+				? 'calypso_editor_page_preview_button_click'
+				: 'calypso_editor_post_preview_button_click';
 			recordEvent( eventLabel );
+			analytics.tracks.recordEvent( eventTracksLabel );
```

Comment:
oops, let's not use `lib/analytics` - it's low-level code meant to be used only by our analytics Redux middleware